docs/script.js

```diff
@@ -0,0 +1,93 @@
+const navMenuButton = document.getElementById("coffee-menu");
+const menuButton = document.getElementById("button-icon-burger");
+const navModal = document.querySelector(".nav-modal");
```

Comment:
maybe add id to it to stick to one method? optional.

Reply:
Done.